mmv1/templates/terraform/examples/dlp_job_trigger_bigquery_row_limit_percentage.tf.erb

```diff
@@ -0,0 +1,37 @@
+resource "google_data_loss_prevention_job_trigger" "<%= ctx[:primary_resource_id] %>" {
+	parent = "projects/<%= ctx[:test_env_vars]['project'] %>"
+	description = "Description"
+	display_name = "Displayname"
+
+	triggers {
+		schedule {
+			recurrence_period_duration = "86400s"
+		}
+	}
+
+	inspect_job {
+		inspect_template_name = "fake"
+		actions {
+			save_findings {
+				output_config {
+					table {
+						project_id = "project"
+						dataset_id = "dataset"
+					}
+				}
+			}
+		}
+		storage_config {
+			big_query_options {
+				table_reference {
+				    project_id = "project"
+				    dataset_id = "dataset"
+				    table_id = "table_to_scan"
+				}
+
+				rowsLimitPercent = 50
+				sampleMethod = "RANDOM_START"
```

Comment:
These field names don't match the terraform field names in the generated configurations. Thus the test is failing.

Could you convert these to camelcase?

Reply:
Ah thanks, will change now. Assuming you mean change the fields in the test to snake case? Been having difficulty running the tests as doesn't seem to be picking up my GCP credentials despite the relevant env vars being set. Running them using the make-in-container script

Reply:
sorry, correct. snake case !